src/mlpack/core/util/sfinae_utility.hpp

```diff
@@ -248,7 +248,6 @@ struct NAME                                                                  \
  *
  * @param METHOD The name of the method to check for.
  * @param NAME The name of the struct to construct.
- * @param MAXN The maximum number of additional arguments.
```

Comment:
I don't believe MAXN exists here or below, but I'm pointing them out in case I've misunderstood.

Reply:
I think once upon a time, `MAXN` was what `7` is now.  But anyway, I think this is fine. :)